templates/zerver/help/index.md

```diff
@@ -127,7 +127,7 @@ as a “**realm**”.
 # Administering a Zulip organization
 
 ## Organization Settings
-* Change your organization's name
+* [Change your organization's name](/help/change-your-org-name)
```

Comment:
please change the link(and the name of file) to `change-your-organizations-name`